### ISSUE TITLE
Uninitialised variable in `pbc.df.aft_ao2mo.get_mo_pairs_G`

### ISSUE THREAD

**Issue by obackhouse** [pyscf/pyscf#680]
The `nao` variable is not initialised in `pbc.df.aft_ao2mo.get_mo_pairs_G`:

```
import numpy as np
from pyscf.pbc import gto, scf, df, tools

cell = gto.M(atom='He 1 0 1; He 0 0 1', a=np.eye(3)*3, mesh=[10,]*3, basis='sto3g')
rhf = scf.KRHF(cell)
rhf.with_df = df.AFTDF(cell)
rhf.kpts = cell.make_kpts([1,1,2])
rhf.conv_tol = 1e-6
rhf.run()

kpts = np.zeros((4,3))

Lij = rhf.with_df.get_mo_pairs((rhf.mo_coeff[0], rhf.mo_coeff[0]), kpts=kpts[:2], q=kpts[1]-kpts[0], compact=False)
```

```
Traceback (most recent call last):
  File "code/kmp2/bug.py", line 13, in <module>
    Lij = rhf.with_df.get_mo_pairs((rhf.mo_coeff[0], rhf.mo_coeff[0]), kpts=kpts[:2], q=kpts[1]-kpts[0], compact=False)
  File "/home/olli/psi4conda/envs/py37/lib/python3.7/site-packages/pyscf/pbc/df/aft_ao2mo.py", line 289, in get_mo_pairs_G
    pqk = (pqkR + pqkI*1j).reshape(nao,nao,-1)
NameError: name 'nao' is not defined
```

I assume this is supposed to be initialised as `nao = mo_coeffs[0].shape[0]`.

**Comment by obackhouse** [comment]
As well as initialising `nao`, I believe the line flagging the error needs to be:
`pqk = lib.unpack_tril(pqkR + pqkI*1j, axis=0).reshape(nao,nao,-1)`
as the loop returns DF blocks with `aosym='s2'`.

**Comment by sunqm** [comment]
Fixed in commit 64e595a8e48a15fbef3f69c2fdce8e748dd390ce

**Comment by obackhouse** [comment]
@sunqm did you see my comment above as well about `lib.unpack_tril` because of `aosym='s2'`? Is this not true?

**Comment by sunqm** [comment]
Sorry didn't notice your following message. Bug fixed in fa93605ead443653973cc023875dbcff1527e781
